src/components/text/textTypes.js

```diff
@@ -0,0 +1,75 @@
+// @flow strict
+export type TextTypeType =
+  |'span'
+  | 'p'
+  | 'h1'
+  | 'h2'
+  | 'h3'
+  | 'h4'
+  | 'h5'
+  | 'h6'
+  | 'div'
+  | 'label'
+  | 'a';
+
+export type TextSizeType =
+  | 'xsmall'
+  | 'small'
+  | 'normal'
+  | 'large'
+  | 'xlarge'
+  | 'xxlarge';
+
+export type TextColorType =
+  | 'default'
+  | 'white'
+  | 'gray'
+  | 'gray-secondary'
+  | 'mint-dark'
+  | 'mint'
+  | 'peach-dark'
+  | 'peach'
+  | 'mustard'
+  | 'blue-dark'
+  | 'blue'
+  | 'blue-secondary'
+  | 'blue-secondary-light';
+
+export type TextWeightType =
+  |'regular'
```

Comment:
some whitespaces missing here and there 👇

Reply:
fixed